ios/SpectrumKit/SpectrumKit/Configuration/FSPConfigurationJpeg.h

```diff
@@ -20,7 +20,7 @@ NS_SWIFT_NAME(ConfigurationJpeg)
                     useProgressive:(BOOL)useProgressive
                    useOptimizeScan:(BOOL)useOptimizeScan
          useCompatibleDCScanOption:(BOOL)useCompatibleDCScanOption
-                 usePSNRQuantTable:(BOOL)usePSNRQuantTable;
+                 usePSNRQuantTable:(BOOL)usePSNRQuantTable NS_DESIGNATED_INITIALIZER;
```

Comment:
We previously had the configuration's initialisers defined as `NS_DESIGNATED_INITIALIZER` but ended up removing them before open sourcing as it forces the configuration's default values to be defined twice: once in the Objective-C layer and once in the [Cpp layer](https://github.com/facebookincubator/spectrum/blob/master/cpp/spectrum/Configuration.h#L145). We'd like to avoid that to keep a single source of truth.

AFAIK designated initialisers particularly make sense when an object cannot function without some provided external dependencies or default values, which isn't the case of `FSPConfiguration*` objects. In that regard, their `intitWithXYZ` methods are move convenience initialisers than anything else.

Does that make sense or am I missing something?

Reply:
Yes, your remark is absolutely right! I have also a bad feelings about having two sources of default values, so I will remove changes connected with configurations.